root-files/opt/flownative/php/build/extensions/xdebug/xdebug.sh

```diff
@@ -39,7 +39,7 @@ extensions_xdebug_runtime_packages() {
 #
 extensions_xdebug_url() {
     if [[ "${PHP_VERSION}" =~ ^8 ]]; then
-        echo "https://xdebug.org/files/xdebug-3.1.1.tgz"
+        echo "https://xdebug.org/files/xdebug-3.2.0alpha3.tgz"
     else
```

Comment:
Maybe create another case, so that we can still use a stable version of XDebug for PHP 8.1 and only use 3.2.0alpha3 with PHP 8.2?

Reply:
Yeah, could do that… Then again, I'd expect this to be merged later, hopefully xdebug 3.2.0 is stable by then. 🤔

Reply:
Xdebug 3.2.0RC2 is out: https://xdebug.org/announcements/2022-11-10

Reply:
I know, see https://github.com/flownative/docker-php/commit/26596c6163dff31ae088bc990302ace3322048e5 – this is an old discussion…